src/variables/body/search.tsx

```diff
@@ -93,16 +96,25 @@ const useOutsideClick = (ref: any, callback: any) => {
 
 const ScopeMenuComponent = ({ model }: { model: Variables.IModel }) => {
   const [toggleState, setToggle] = useState(false);
+  const [scopes, setScopes] = useState(model.scopes);
   const [scope, setScope] = useState(model.currentScope);
```

Comment:
Do we need both `scope` and `scopes` in the state?

Reply:
Scopes can be changed. For now, we get only Local scope or an empty array.